### ISSUE TITLE
[May not be the bug] Just want few help from youe team relate with docker

### ISSUE THREAD

**Issue by fei15115** [jammsen/docker-palworld-dedicated-server#104]
### Have you read the Important information text above

- [X] Yes i did

### Current behavior

When I restart the linux server, I find no container is running or exists.
So I tried to rebuild the container with docker-compose, but I got some error with the port already get used.
Then I find the game server is already runing with no any reference(no docker container).
![image](https://github.com/jammsen/docker-palworld-dedicated-server/assets/37013076/465d3a9e-e050-47ec-967b-102bfb4b62d0)

### Desired behavior

I'm the new guy for the docker-compose, Sorry for the bother.
So should I start the container every time?
Why I could not find the comtainer in docker?

### Software setup

- OS:linux 20.03
- Docker:25.0.0

### Hardware setup

- vCPU: 4c
- RAM: 16g
- Disk:64G

**Comment by jammsen** [comment]
The standard is that Docker creates containers and they have something called a restart-policy, so when you reboot or docker-daemon restart they know what they are doing. But as you wrote it, just container is lost, are you sure about that?

What happens when you du "docker ps -a" are there exited containers?

If its all clear after a reboot, something is mixed up with your docker and linx install, cant really help with that, try google and docker-support forum then.

**Comment by fei15115** [comment]
Thank you for your quick reply,
That also confuses me. "ps -a " shows the same:
![image](https://github.com/jammsen/docker-palworld-dedicated-server/assets/37013076/15df0da1-b475-485d-a6ab-5270a7364e18)
Maybe I need to reinstall the Linux into my exsi system.If I want to save the game record, I need to expose all the file below the "Pal/saved" right ? Is there any missing file ?

**Comment by jammsen** [comment]
> Maybe I need to reinstall the Linux into my exsi system.

Yeah try a new VM, with a fresh install of ubuntu-server or debian with only docker 25.0.0 and start again

> If I want to save the game record, I need to expose all the file below the "Pal/saved" right ? Is there any missing file ?

Im not sure about that, my backup script saves `${GAME_PATH}/Pal/Saved` and all of its content. Just like the Unreal Engine way is. Maybe thats to much, but i dont care for min-maxing savegames, that way i dont miss anything 😝

**Comment by jammsen** [comment]
@fei15115 did you solve it?

**Comment by fei15115** [comment]
Hi， Sorry for the delay response.
I have already reinstall the linux, and it works this time.
I think it might be something mix into the docker deamon or something I don't know.
greate docker and help,Thanks!
